protocols/S3/stationA_protocol3_lysates_S3.ot2.apiv2.py

```diff
@@ -98,23 +96,41 @@
     }
 
 # Function definitions
-def run_info(start, end, parameters = dict()):
```

Comment:
I need the parameters to be passed as arguments, because I want this function to be the same for all protocols, and each protocols have different parameters